### PR TITLE
hello world

### PR DESCRIPTION
**Pull request by victoriCrediday**
blah blah

### CODE REVIEW

File: staff/victori-murillo/hello-world/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Hello World!</title>
+</head>
+<body>
+  <h1>Hello, World!</h1>
```
Comment (victoriCrediday) [inline staff/victori-murillo/hello-world/index.html]: blah

File: staff/victori-murillo/hello-world/style.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body {
+  background-color: yellow;
+}
+
+h1 {
+  cursor: pointer;
```
Comment (victoriCrediday) [inline staff/victori-murillo/hello-world/style.css]: blah blah

File: staff/victori-murillo/hello-world/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
```
Comment (ericaig) [inline staff/victori-murillo/hello-world/index.html]: New comment:: style.css

File: staff/victori-murillo/hello-world/index.html (hunk 1 above)
Comment (ericaig) [inline staff/victori-murillo/hello-world/index.html]: Hola, Mundo!

File: staff/victori-murillo/hello-world/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  <title>Hello World!</title>
+</head>
+<body>
+  <h1>Hello, World!</h1>
+
+  <p>Hll Wld!</p>
```
Comment (ericaig) [inline staff/victori-murillo/hello-world/index.html]: hey, this text should be `Hello, World!`

**Pull request merged**